sleap_nn/training/model_trainer.py

```
@@ -2,14 +2,17 @@
 
 from pathlib import Path
 from typing import Optional, List
+import functools
 import time
 from torch import nn
 import os
+import shutil
 import torch
 import sleap_io as sio
 from torch.utils.data import DataLoader
```

Ruff

`torch.utils.data.DataLoader` imported but unused

Remove unused import: `torch.utils.data.DataLoader`

(F401)